[2025-10-02T06:24:17.357+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T06:24:17.365+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentiment140_end2end_single.sf_check_label_dist manual__2025-10-02T06:23:19.658047+00:00 [queued]>
[2025-10-02T06:24:17.368+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentiment140_end2end_single.sf_check_label_dist manual__2025-10-02T06:23:19.658047+00:00 [queued]>
[2025-10-02T06:24:17.368+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-02T06:24:17.372+0000] {taskinstance.py:2330} INFO - Executing <Task(SQLExecuteQueryOperator): sf_check_label_dist> on 2025-10-02 06:23:19.658047+00:00
[2025-10-02T06:24:17.376+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=917) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T06:24:17.377+0000] {standard_task_runner.py:63} INFO - Started process 921 to run task
[2025-10-02T06:24:17.377+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sentiment140_end2end_single', 'sf_check_label_dist', 'manual__2025-10-02T06:23:19.658047+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/edaproject.py', '--cfg-path', '/tmp/tmpv4a91qzd']
[2025-10-02T06:24:17.379+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask sf_check_label_dist
[2025-10-02T06:24:17.399+0000] {task_command.py:426} INFO - Running <TaskInstance: sentiment140_end2end_single.sf_check_label_dist manual__2025-10-02T06:23:19.658047+00:00 [running]> on host b4099255ca46
[2025-10-02T06:24:17.430+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentiment140_end2end_single' AIRFLOW_CTX_TASK_ID='sf_check_label_dist' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T06:23:19.658047+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T06:23:19.658047+00:00'
[2025-10-02T06:24:17.431+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T06:24:17.437+0000] {sql.py:276} INFO - Executing: 
            SELECT LABEL, COUNT(*) AS CNT
            FROM YOUR_DB.PUBLIC.SENTIMENT140_CLEAN
            GROUP BY 1
            ORDER BY 1;
        
[2025-10-02T06:24:17.441+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-10-02T06:24:17.444+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-10-02T06:24:17.445+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-02T06:24:17.445+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-02T06:24:18.871+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:18.872+0000] {sql.py:470} INFO - Running statement: SELECT LABEL, COUNT(*) AS CNT
            FROM YOUR_DB.PUBLIC.SENTIMENT140_CLEAN
            GROUP BY 1
            ORDER BY 1;, parameters: None
[2025-10-02T06:24:19.295+0000] {cursor.py:1149} INFO - Number of results in first chunk: 2
[2025-10-02T06:24:19.295+0000] {sql.py:479} INFO - Rows affected: 2
[2025-10-02T06:24:19.299+0000] {snowflake.py:407} INFO - Rows affected: 2
[2025-10-02T06:24:19.299+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6f60-0306-908d-0000-00131e678171
[2025-10-02T06:24:19.438+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:19.438+0000] {connection.py:762} INFO - closed
[2025-10-02T06:24:19.510+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2025-10-02T06:24:19.585+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T06:24:19.616+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sentiment140_end2end_single, task_id=sf_check_label_dist, run_id=manual__2025-10-02T06:23:19.658047+00:00, execution_date=20251002T062319, start_date=20251002T062417, end_date=20251002T062419
[2025-10-02T06:24:19.659+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T06:24:19.668+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-02T06:24:19.669+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
